include/d/actor/d_a_dk.h

```diff
@@ -1,23 +1,65 @@
 #ifndef D_A_DK_H
 #define D_A_DK_H
 
+#include "d/d_bg_s_acch.h"
+#include "d/d_particle.h"
 #include "f_op/f_op_actor.h"
+#include "m_Do/m_Do_hostIO.h"
 
-struct tail_s {
-    
-};
+class tail_s {
+    public:
+    /* 0x000 */ J3DModel* models[9];
+    /* 0x024 */ cXyz field_0x024[10];
+    /* 0x09C */ csXyz field_0x09C[10];
+    /* 0x0D8 */ cXyz field_0x0D8[10];
+    /* 0x150 */ cXyz field_0x150[2];
+    /* 0x168 */ csXyz field_0x168;
+    /* 0x16E */ u8 field_0x16E[0x170 - 0x16E];
+    /* 0x170 */ cXyz field_0x170;
+};  // Size: 0x17C
 
 class dk_class : public fopAc_ac_c {
 public:
-    /* Place member variables here */
-};
 
-class daDk_HIO_c {
+    dk_class() : field_0xACC(1, 1, 0, 0) {
+
+    }
+
+    /* 0x290 */ u8 field_0x290[0x2AC - 0x290];
+    /* 0x2AC */ request_of_phase_process_class mPhs;
+    /* 0x2B4 */ u8 field_0x2B4;
+    /* 0x2B5 */ u8 field_0x2B5[0x2B8 - 0x2B5];
```

Comment:
```suggestion
```
Same with the rest of these padding fields in this header, except for field_0xAC4 and field_0xAEC

Reply:
Done, I always forget to do this after exporting the classes from ghidra.